developers/advanced/advanced-pinot-setup.md

```diff
@@ -386,6 +413,12 @@ bin/pinot-admin.sh AddTable \
 {% endtab %}
 {% endtabs %}
 
+### Use `sortedColumn` with streaming tables
+
+For [streaming](#streaming-table-creation) tables, you can use a sorted index with `sortedColumn`. This will sort data when generating segments as the segment is committed. See [Real-time tables](../../basics/indexing/forward-index.md#real-time-tables) for more information.
```

Comment:
```suggestion
For [streaming](#streaming-table-creation) tables, you can use a sorted index with `sortedColumn` to sort data when generating segments as the segment is committed. See [Real-time tables](../../basics/indexing/forward-index.md#real-time-tables) for more information.
```

Reply:
does committed here mean the same as created? if so, perhaps we say that, if not, lets clarify the distinction